scripts/default_worker.py

```diff
@@ -2,10 +2,6 @@
 import argparse
 import numpy as np
 
-import ray.array.remote as ra
-import ray.array.distributed as da
-import example_functions
-
```

Comment:
Yeah, if needed, these will be imported on the driver (and subsequently exported to the workers) as opposed to being imported on the workers directly.